app/views/users/_profile.html.erb

```diff
@@ -58,7 +58,7 @@
 
     <div class="crayons-field crayons-field--checkbox">
       <%= f.check_box "profile[display_email_on_profile]",
-                      checked: profile.display_email_on_profile,
+                      checked: @user.setting.display_email_on_profile,
```

Comment:
I noticed this as a leftover from the user settings work and it got migrated over together with the commit I cherry-picked from another branch.